documentation/example-file-usage/mqtt-example-file-usage.md

```diff
@@ -0,0 +1,136 @@
+
+4. Enter in the following command to install/check Python 3:
+
+   ```powershell
+   python3
+   ```
+
+   Now, one of two things will happen: 
+
+   - The first possibility is that the Microsoft Store will open and prompt you to install Python 3.10. You can do that and that's a perfectly fine way to install Python 3 (it's the method I used) or you can go to the [Python website](https://www.python.org/downloads/) and install it from there. Either way works, do whichever you prefer. Enter in the above command again after and you should see what's in the next bullet.
+
+   - The other possibility is that you get a message similar to this:
+
+     ```
+     Python 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)] on win32
+     Type "help", "copyright", "credits" or "license" for more information.
+     >>> 
+     ```
+
+     This means you already have Python 3 installed and are good to go on to the next step. Enter in the following command to exit out of the Python 3 runtime environment.
+
+     ```python
+     quit()
+     ```
+
+5. Enter in the following command to install the required pip packages:
+
+   ```
+   pip3 install -r requirements.txt
+   ```
+
+   If required, enter `Y` to all installation prompts.
+
+### MacOS (Python 3, and pip packages)
+
+1. Install the AnkerMake slicer from [their website](https://www.ankermake.com/software). Make sure you open it and login, then you can close the app.
+
+2. Using `git` or GitHub Desktop, clone this repository into a location of your choice and then navigate to that location in Finder.
+
+3. Hold down the "Control" key and click the folder in the path bar, then choose "Open in Terminal". (If you don’t see the path bar at the bottom of the Finder window, choose View > Show Path Bar)
+
+4. Install Python3 from the [Python website.](https://www.python.org/downloads/macos/)
+
+5. Enter in the following command to install the required pip packages:
+
+   ```
+   pip3 install -r requirements.txt
+   ```
+
+   If required, enter `Y` to all installation prompts.
+
+
+
+## Extracting your Auth Token
+
+Your Auth Token is the "key to the kingdom" so-to-speak. The M5 uses this to verify who can connect to it over the internet (or LAN) to make sure it's actually you or someone you've shared your printer with (although, we haven't done any testing with shared printers so that's more of an educated guess). You're printer will refuse any connections that don't present your unique Auth Token, so let's get that out for future use.
+
+1. Navigate to wherever you cloned this repository to. Open the "examples" folder and open a terminal window there, just like in the previous section.
+
+2. Type in the following command:
+
+   ```bash
+    python3 extract-auth-token.py -a
+   ```
+
+   You should get back a long sequence of numbers and letters, that's your Auth Token. Copy that to the clipboard for use in the next section.
```

Comment:
Maybe add an example with `-f /path/to/login.json` ? That would help people with non-standard install locations.